Remove the 'have_library()' check. It breaks cross compilation.
The checked for libxslt and libexslt are pulled in via makedepends
and therefore do exist, anyway.

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -157,20 +157,8 @@
     $config{LIBS} .= $::is_Win32 ? '' :' -lm';
 }
 
-if (!have_library($::is_Win32 ? "libxslt" : "xslt")) {
-    print STDERR <<DEATH;
-libxslt not found
-Try setting LIBS and INC values on the command line
-Or get libxslt and libxml2 from
-  http://www.libxml.org/
-If you install via RPMs, make sure you also install the -devel
-RPMs, as this is where the headers (.h files) are.
-DEATH
-exit 0; # 0 recommended by http://cpantest.grango.org (Notes for CPAN Authors)
-}
-
 
-if (have_library($::is_Win32 ? "libexslt" : "exslt")) {
+if ($::is_Win32 ? "libexslt" : "exslt") {
     if (! $HAVE_USER_DEFINED) {
         my $exslt_defaults = $::is_Win32 ?
         ($ENV{ACTIVEPERL_MINGW} ? q/-llibexslt.lib/ : q/-llibexslt/) :
